core/BUILD.bazel

```diff
@@ -16,6 +16,17 @@ java_library(
     ],
 )
 
+# javax.annotation.Generated is not included in the default root modules in 9,
+# see: http://openjdk.java.net/jeps/320.
+java_import(
+    name = "javax_annotation",
```

Comment:
Technically, having this in core seems a bad idea, but we don't have a third_party/ yet, so this seems good enough for now.

Reply:
Ack.

Reply:
Maybe in `stub` is more appropriate than `core`, although not as good as `third_party`.

Reply:
I moved the new javax_annotation rule to stub package instead of core. Done.